static/includes/faq-ai.md

```diff
@@ -0,0 +1,23 @@
+**Does Aiven AI Optimizer mask/obfuscate my queries?**
+
+Yes, Aiven AI Optimized provides a non-intrusive solution to optimize your
+database performance without compromising sensitive data access.
+It achieves this by gathering information on schema structure, database statistics, and
+other signals to detect potential performance problems and offer optimization
+recommendations, without requiring credentials or access to the actual data in
+the database.
+
+To address the possibility of slow query logs containing sensitive data, Aiven
+offers data masking capabilities that replaces sensitive parameters within
+queries with question marks (`?`). Data masking is enabled by default.
+
+:::note
+The masking option is not available for
+the Standalone SQL query optimizer yet.
+:::
+
+**Is there an option to ask Aiven not to keep my SQL queries?**
```

Comment:
@ArthurFlag - remove this question, as we are working on a new checkbox that will allow the user to select 'don't save my query'. you can maybe add this specific question in 3-4 weeks when feature will go live.
(jira: https://aiven.atlassian.net/browse/EVERSQL-418)